pkgs/development/python-modules/pycritty/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder, pyyaml }:
+
+buildPythonPackage rec {
+  pname = "pycritty";
+  version = "0.3.5";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1lrmd4a1ps3h9z0pndfjfrd2qa7v3abd6np75fd2q2ffsqv7ar6x";
+  };
+
+  # The package does not include any tests to run
+  doCheck = false;
+
+  propagatedBuildInputs = [ pyyaml ];
+
+  postPatch = ''
+    # remove custom install
+    substituteInPlace setup.py \
+      --replace "'install': PostInstallHook," ""
+  '';
+
+  meta = with lib; {
+    description = "A CLI tool for changing your alacritty configuration on the fly.";
```

```suggestion
    description = "A CLI tool for changing your alacritty configuration on the fly";
```

No period at the end, see https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes

---

@fabaff Thanks for the tip - I removed the trailing period. I forgot to check the meta attribute requirements; my bad.